README.md

```diff
@@ -237,6 +237,19 @@ data already loaded.
   rails server
   ```
 
+### API documentation
+
+Our API documentation is in the `openapi` directory. It follows the OpenAPI
```

Comment:
What about changing `openapi` to a link, like [openapi](https://github.com/solidusio/solidus/tree/master/openapi)?

Reply:
@spaghetticode done!